recipes/cookiecutter-django/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "cookiecutter-django"%}
+{% set version = "1.11.9" %}
+{% set sha256 = "1d4b2fd48e49d0c78b90a2f47c78db191e5ed90bf06b38deb414022e4bda2476" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+    - virtualenv
+  run:
```

Comment:
Where are all of these dependencies defined? I only see a few in:

https://github.com/pydanny/cookiecutter-django/blob/master/requirements.txt

Reply:
@synapticarbors : There are four sets of dependencies that are listed in https://github.com/pydanny/cookiecutter-django/tree/master/%7B%7Bcookiecutter.project_slug%7D%7D/requirements